### DISCUSSION TITLE
Align the table

### DISCUSSION THREAD

**Discussion by Kiber-pank** [dolanmiu/docx#1500] [Q&A] [answered]
Please tell me how to align the table to the center of the page
I use docs on the client side

**Comment by Icaro-Lima** [comment, marked as answer]
Have you tried adding it inside a `Paragraph` with `alignment:` set to `AlignmentType.CENTER`?

**Reply by Kiber-pank** [reply]
> Have you tried adding it inside a `Paragraph` with `alignment:` set to `AlignmentType.CENTER`?

should the whole table be placed in a paragraph?

**Reply by QWERTY-WINRAR** [reply]
I'd like to know too, because i've tried to put a `Table` inside a `Paragraph`, and now the new files cannot be opened...

```javascript
let table1 = [];
  table1.push(
    new docx.TableCell({
      children: [new docx.Paragraph("XXXXXXX")]
    })
  );
  table1.push(
    new docx.TableCell({
      children: [new docx.Paragraph("YYYYYYYYY")]
    })
  );
  table1.push(
    new docx.TableCell({
      children: [new docx.Paragraph("ZZZZZZZZZZ")]
    })
  );

  const parArray = [];
  parArray.push(
    new docx.Paragraph({
      children: [
        new docx.Table({
          rows: [
            new docx.TableRow({
              children: table1
            })
          ],
          width: {
            size: 100,
            type: docx.WidthType.PERCENTAGE
          }
        })
      ]
    })
  );
``` 

Also, i'll be happy to know if there's any way to make the new `TableCells` to be created from the right side, instead from the left.
Thank you!
